docs/en/manage/tuning-for-cloud-cost-efficiency.md

```diff
@@ -58,6 +58,15 @@ As an example, this is how you can do that within a JDBC connection string when
 ```
 
 
+:::note Automatic deduplication is disabled when using asynchronous inserts
+Manual batching (see [section above](#ingest-data-in-bulk))) has the advantage that it supports the [built-in automatic deduplication](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replication/) 
+of table data if (exactly) the same insert statement is send mutliple times to ClickHouse Cloud, 
+for example because of an automatic retry in a client software because of some temporarily network connection issues.
+
+Asynchronous inserts don't support this built-in automatic deduplication of table data in case of insert statement retries.
```

Comment:
I think maybe drop the `in case of insert statement retries`.  I think dedup is not done no matter the reason if using async.

Reply:
Makes sense, let's drop.